VTEX - Checkout API.json

```diff
@@ -19338,6 +19359,18 @@
                         "description": "Configures reCAPTCHA validation for the account, defining in which situations the shopper will be prompted to validate a purchase with reCAPTCHA. Learn more about [reCAPTCHA validation for VTEX stores](https://help.vtex.com/tutorial/recaptcha-no-checkout--18Te3oDd7f4qcjKu9jhNzP)\n\r\n\rPossible values are:\n\r- `\"never\"`: no purchases are validated with reCAPTCHA.\n\r- `\"always\"`: every purchase is validated with reCAPTCHA.\n\r- `\"vtexCriteria\"`: only some purchases are validated with reCAPTCHA in order to minimize friction and improve shopping experience. VTEX’s algorithm determines which sessions are trustworthy and which should be validated with reCAPTCHA. This is the recommended option.",
                         "default": "vtexCriteria"
                     },
+                    "requiresLoginToPlaceOrder": {
+                        "type": "boolean",
+                        "description": "Indicates whether authentication is required for completing purchases."

```

Comment:
É interessante indicar o comportamento para cada valor. Ex: "When set as `false`, it indicates xxx, when set as `true`, it indicates yyy". E se tiver alguma recomendação de segurança para definir como `true`, mencionar também.